lighthouse-core/audits/font-display.js

```diff
@@ -25,11 +25,13 @@ const UIStrings = {
     'webfonts are loading. ' +
     '[Learn more](https://web.dev/font-display/).',
   /**
-   * @description A warning message that is shown when Lighthouse couldn't automatically check some of the page's fonts and that the user will need to manually check it.
-   * @example {https://font.cdn.com/} fontURL
+   * @description [ICU Syntax] A warning message that is shown when Lighthouse couldn't automatically check some of the page's fonts, telling the user that they will need to manually check them.
+   * @example {https://font.cdn.com/} fontOrigin
    */
-  undeclaredFontURLWarning: 'Lighthouse was unable to automatically check the font-display value ' +
-    'for the following URL: {fontURL}.',
+  undeclaredFontOriginWarning:
+    'Lighthouse was unable to automatically check the {fontCountForOrigin, plural, ' +
+    '=1 {`font-display` value for the following origin: {fontOrigin}.} ' +
```

Comment:
nit: maybe drop both usages of `following`?

Reply:
> nit: maybe drop both usages of `following`?

yeah, I'm not super in love with the warning text overall, but it's rarely seen enough that I don't have any interest in trying out a whole new sentence. Just dropping 'following' sounds good :)